conf/defaults.yaml

```diff
@@ -0,0 +1,25 @@
+# This is example of USM QE configuration file.
+
+# All options in `usmqepytest` section will be loaded during pytest start
+# so that it will be available as if it were specified directly in pytest.ini
+# (see usmqe/usmqeconfig.py for details).
+usmqe:
+  log_level: DEBUG
+  username: admin
+  password: adminuser
+  ca_cert: conf/tendrl_ca.crt
+  gluster_role: gluster-servers
+  brick_name: brick
+  # this is just example of ldap setup, it's not currently used for anything
+  ldap:
```

Comment:
If I understand the overall approach well, I think, that the `defaults.yaml`should not contain `ldap` section, which is (as mentioned in the comment) just not used example of syntax. I think, it should contains only actually used variables. If anywhere, I would put the example section to the `example-usm1.yaml` file, which is not supposed to be directly used.

Reply:
I agree to remove configuration related to ldap but I would do it in a different PR where will be removed also python code related to ldap.

Reply:
ok

Reply:
ack